<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionAnswerForm</class>
 <widget class="QWidget" name="QuestionAnswerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>493</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Диагностика живых объектов</string>
  </property>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Далее</string>
   </property>
  </widget>
  <widget class="QPushButton" name="visualizeButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Визуализировать</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>700</width>
     <height>71</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 225, 225)</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.25pt;&quot;&gt;Если Вы не уверены в достоверности введенной информации, воспользуйтесь кнопкой &amp;quot;Инструменты&amp;quot;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nefactorsButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>700</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Инструменты</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tempInfoButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>90</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Временная информация</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="questionEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>700</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>700</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="questionLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>700</width>
     <height>61</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Укажите симптом</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="answerLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>700</width>
     <height>61</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Укажите значение симптома</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
